fairlearn/reductions/_exponentiated_gradient/_lagrangian.py

```diff
@@ -56,39 +56,39 @@ def __init__(self, X, sensitive_features, y, estimator, constraints, eps, B, opt
         self.last_linprog_n_hs = 0
         self.last_linprog_result = None
 
-    def _eval_from_error_gamma(self, error, gamma, lambda_vec):
+    def _eval(self, h, lambda_vec):
         """Return the value of the Lagrangian.
```

Comment:
This is a docstring which could use some expansion :-)

Reply:
Sure, but that's not the point of this PR.